regtests/bin/matrix.base

```diff
@@ -820,20 +809,7 @@
       echo "$rtst -s PR3_UQ_MPI              -w work_PR3_UQ_MPI                  -f -p $mpi -n $np $ww3 ww3_tp2.13" >> matrix.body
       echo "$rtst -s MPI                     -w work_PR3_UQ_MPI                  -f -p $mpi -n $np $ww3 ww3_tp2.15" >> matrix.body
       echo "$rtst -s MPI                     -w work_MPI_5km         -g 5km      -f -p $mpi -n $np $ww3 ww3_tp2.15" >> matrix.body
-      echo "$rtst -s MPI        -i input_rho -w work_PR3_UQ_RHO_MPI -r ww3_grid  -f -p $mpi -n $np $ww3 ww3_tp2.15" >> matrix.body
```

Comment:
Same comment above applies here.

Reply:
the PR description now includes:
"Other changes included here:
Includes the correction of the typo in OASACM switch (L2264 in ww3_shel.ftn).
Changes in matrix.base to avoid running all the ww3_tp2.15 programs separately"